tools/fiptool/fiptool.c

```diff
@@ -52,34 +52,6 @@
 #define OPT_PLAT_TOC_FLAGS 1
 #define OPT_ALIGN 2
 
-static image_desc_t *lookup_image_desc_from_uuid(const uuid_t *uuid);
```

Comment:
This boils down to preferences.  TF is not consistent in this regard either, just check drivers/io/io_block.c.
I will let others voice their opinion and we can decide later.